Core/TermsOfService.swift

```diff
@@ -20,125 +20,84 @@
 import Foundation
 
 public struct TermsOfService: Decodable {
-
-    public init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: CodingKeys.self)
-        self.classification = try? container.decode(Classification.self, forKey: .classification)
-        self.score = try container.decode(Int.self, forKey: .score)
-        let matchContainer = try container.nestedContainer(keyedBy: CodingKeys.Match.self, forKey: .match)
-        self.goodReasons = try matchContainer.decodeIfPresent([String].self, forKey: .good) ?? []
-        self.badReasons = try matchContainer.decodeIfPresent([String].self, forKey: .bad) ?? []
```

Comment:
This seemed nasty to cope with the class being a string or false. I changed the source JSON to make false null instead and made the struct match the json format more directly.

Reply:
I don't think the json should be changed, see my comment on the DuckDuckGoTests/MockFiles/tosdr.json file

Reply:
Reverted the tosdr files and reverted the parsing logic